cmd/cdi-controller/controller.go

```diff
@@ -369,8 +369,12 @@ func getTokenPrivateKey() *rsa.PrivateKey {
 func registerMetrics() {
 	metrics.Registry.MustRegister(controller.IncompleteProfileGauge)
 	controller.IncompleteProfileGauge.Set(-1)
+	metrics.Registry.MustRegister(controller.IncompleteProfileGaugeVec)
+	metrics.Registry.MustRegister(controller.DefaultStorageClassesGauge)
 	metrics.Registry.MustRegister(controller.DefaultVirtStorageClassesGauge)
+	metrics.Registry.MustRegister(controller.DefaultStorageClassWithSmartCloneAndRWXGauge)
```

Comment:
Just following up on offline discussion:
Maybe make `DefaultStorageClassWithSmartCloneAndRWXGauge` just about virt storage classes? Even then I fear we can't bug people who have some basic storage solution. But as long as we see this in telemetry we can adjust ourselves

Reply:
I tend to agree.